massgov/mayflower
# patternlab/DP-26999 feedback fix invalid error msg

To test: 
1. Go to https://mayflower.digital.mass.gov/patternlab/b/patternlab/DP-26999-feedback-fix-invalid-error-msg/?p=organisms-mass-feedback
2. Select Yes and type in a phone number
3. Switch to No
4. Then switch back to Yes
5. The text area should be empty and validation should reset properly

Test this for both scenarios 
https://mayflower.digital.mass.gov/patternlab/b/patternlab/DP-26999-feedback-fix-invalid-error-msg/?p=organisms-mass-feedback
https://mayflower.digital.mass.gov/patternlab/b/patternlab/DP-26999-feedback-fix-invalid-error-msg/?p=organisms-mass-feedback-alert
